index.html

```diff
@@ -211,6 +234,11 @@ <h2>Example</h2>
 
     <h2>IANA Considerations</h2>
     <section id="media-type">
+      <div class="issue">
+        <h3>Correct location of media type registration</h3>
```

Comment:
can we remove this issue, I don't think we need an RFC to request a media type, this section is based on: https://www.w3.org/TR/activitystreams-core/#iana

Reply:
+1